tests/cache/tests.py

```diff
@@ -621,6 +621,16 @@ def test_cull(self):
     def test_zero_cull(self):
         self._perform_cull_test('zero_cull', 50, 19)
 
+    def test_cull_delete_when_table_empty(self):
```

Comment:
I'd just rename this test to avoid references to _table_ since it doesn't make sense for non-database backends.

```suggestion
    def test_cull_delete_when_store_empty(self):
```

Reply:
Make sense, done! Thanks again!